### PR TITLE
Add Rambler/TOP-100 analytics vendor to docs

### LINKED ISSUE #630: Add description for new analytics vendor

### PR DESCRIPTION
**Pull request by arealit**
Fixes #630

### CODE REVIEW

**Comment by bpaduch** [comment]
Related info: 
- Analytics vendor code added in https://github.com/ampproject/amphtml/pull/10328
- ITI: https://github.com/ampproject/amphtml/issues/10327

File: content/docs/guides/analytics_amp/analytics-vendors@ja.md [hunk 1]
```diff
@@ -230,6 +230,12 @@ type 属性の値: `snowplow`
 
 この値を指定すると、Snowplow Analytics を利用できるようになります。Snowplow Analytics の利用を有効にする方法について詳しくは、[github.com/snowplow/snowplow/wiki](https://github.com/snowplow/snowplow/wiki/Google-AMP-Tracker) をご覧ください。
 
+### Rambler/TOP-100
+
+type 属性の値: `top100`
+
+この値を指定すると、Rambler/TOP-100 を利用できるようになります。設定の詳細については、[top100.rambler.ru](https://top100.rambler.ru/docs).
```
Comment (bpaduch) [inline content/docs/guides/analytics_amp/analytics-vendors@ja.md]: Tack on to the end (after the link), just like Webtrekk has: をご覧ください。

File: content/docs/guides/analytics_amp/analytics-vendors@ko.md [hunk 2]
```diff
@@ -229,6 +229,12 @@ toc: true
 
  Snowplow Analytics 지원을 추가합니다. Snowplow Analytics 지원 추가에 관한 자세한 내용은 [github.com/snowplow/snowplow/wiki](https://github.com/snowplow/snowplow/wiki/Google-AMP-Tracker) 에서 확인할 수 있습니다.
 
+### Rambler/TOP-100
+
+속성값 유형: `top100`
+
+ Rambler/TOP-100 지원을 추가합니다. 구성 관련 세부정보는 [top100.rambler.ru](https://top100.rambler.ru/docs).
```
Comment (bpaduch) [inline content/docs/guides/analytics_amp/analytics-vendors@ko.md]: Tack on to the end (after the link), just like Webtrekk has:  에서 확인할 수 있습니다.

File: content/docs/guides/analytics_amp/analytics-vendors@ja.md (hunk 1 above)
Comment (arealit) [inline content/docs/guides/analytics_amp/analytics-vendors@ja.md]: Done

File: content/docs/guides/analytics_amp/analytics-vendors@ko.md (hunk 2 above)
Comment (arealit) [inline content/docs/guides/analytics_amp/analytics-vendors@ko.md]: Done

**Pull request merged**